lib/matplotlib/rcsetup.py

```diff
@@ -1020,6 +1020,10 @@ def _convert_validator_spec(key, conv):
     "polaraxes.grid": validate_bool,  # display polar grid or not
     "axes3d.grid":    validate_bool,  # display 3d grid
 
+    "axes3d.panecolorx":    validate_color,  # 3d background pane
+    "axes3d.panecolory":    validate_color,  # 3d background pane
+    "axes3d.panecolorz":    validate_color,  # 3d background pane
```

Comment:
```suggestion
    "axes3d.xaxis.panecolor":    validate_color,  # 3d background pane
    "axes3d.yaxis.panecolor":    validate_color,  # 3d background pane
    "axes3d.zaxis.panecolor":    validate_color,  # 3d background pane
```
It seems better structured and more futureproof to add the "*axis" level and not only append x,y,z to the name.